bots/api/bot.py

```
@@ -1,11 +1,18 @@
 import os
+import platform
```

Ruff (0.8.2)

`platform` imported but unused

Remove unused import: `platform`

(F401)